docs/rules/no-cb-test.md

```diff
@@ -0,0 +1,26 @@
+# Ensure no `test.cb()` is used
+
+Disallow the use of `test.cb()`.
+
+
+## Fail
+
+```js
+const test = require('ava');
+
+test.cb('some test', t => {
+	t.pass();
+	t.end();
+});
+```
+
+
+## Pass
+
+```js
+const test = require('ava');
+
+test('some test', t => {
+	t.pass();
+});
```

Comment:
```js
test('some test', async t => {
	t.pass();
});
```

would be a better example as it's the alternative we recommend over `test.cb()`.